style.css

```diff
@@ -261,63 +259,21 @@ ul {
   padding-top: 10px;
 }
 
-.language-blocks {
+.languages {
   display: flex;
-  flex-wrap: nowrap;
-  align-items: center;
-  padding: 4px 2px;
-  margin-top: 20px;
-  background: #ebf0ee;
-  color: #3a4a42;
```

Comment:
- [ ] [OPTIONAL] This is very important, but because the last reviewer didn't call it out, I'm making as optional.

- Your design
<img width="468" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/b7a430f3-82b0-4336-a590-3903bcabffbe">

- Figma design
<img width="387" alt="image" src="https://github.com/AlyAtalla/AlyAtalla.github.io/assets/41163571/80ed76b1-c137-4104-ba05-ee0d0573bb18">

- Kindly fix the image width to align with the other items like it is in the Figma design, to match the design 💯.